src/bluesky_stomp/messaging.py

```diff
@@ -230,23 +228,27 @@ def _send_bytes(
         correlation_id: str | None = None,
     ) -> None:
         logging.info(f"SENDING {message} to {destination}")
+        with self.tracer.start_as_current_span(
+            "_send_bytes",
+            attributes={"destination": destination, "message": message},
+            kind=SpanKind.SERVER,
+        ):
+            headers: dict[str, Any] = {"JMSType": "TextMessage"}
+
+            if on_reply is not None:
+                reply_queue = TemporaryMessageQueue(name=str(uuid.uuid1()))
+                headers = {**headers, "reply-to": _destination(reply_queue)}
+                self.subscribe(
+                    reply_queue,
+                    on_reply,
+                    on_error=on_reply_error,
+                )
+            if correlation_id:
+                headers = {**headers, CORRELATION_ID_HEADER: correlation_id}
```

Comment:
```suggestion
            headers: dict[str, Any] = {"JMSType": "TextMessage"}

            if on_reply is not None:
                reply_queue = TemporaryMessageQueue(name=str(uuid.uuid1()))
                headers = ["reply-to"] = _destination(reply_queue)
                self.subscribe(
                    reply_queue,
                    on_reply,
                    on_error=on_reply_error,
                )
            if correlation_id:
                headers[CORRELATION_ID_HEADER] = correlation_id
```

I believe it's my code in the first place, but this stops us making 3 dicts when all we really want to do is add an entry to the same dict.

Reply:
I think the syntax on your line 239 is wrong, did you mean

`headers["reply-to"] = _destination(reply_queue)`

Reply:
yup my bad

Reply:
Done